app/assets/javascripts/rails_db/show_hide_columns.js

```diff
@@ -0,0 +1,54 @@
+$(function(){
+  $(document).foundation();
+
+  $('body').on('click', '.settings-link', function() {
+    $('.rails-db-settings').toggle();
+  });
+
+  $('body').on('click', '.column-name input', function() {
+    table_name = $('h2')[0].innerHTML;

```

Comment:
place in settings link attribute with table name, because what if tomorrow I'll have h3? Or I'll have another h2 on page before this? ALWAYS think how to brake system